spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -124,4 +160,11 @@ public void setMessageConverter(MessageConverter messageConverter) {
 		this.messageConverter = messageConverter;
 	}
 
+	public int getExecutorThreadCount() {
+		return this.executorThreadCount;
+	}
+
+	public void setExecutorThreadCount(int executorThreadCount) {
+		this.executorThreadCount = executorThreadCount;
```

Comment:
This one gave me a bit to think about. I don't really like updating the thread pool setting, since that would mean we'd have to rebuild every existing publisher to use the new settings.

OTOH also not a big fan of having that setting in the constructor. It's what makes more sense, but it could confuse users?

We could have two constructors, one with and one without `executorThreadCount`, which just uses the default. Would that be reasonable?